src/main/java/com/icthh/xm/tmf/ms/loyalty/service/LoyaltyAccountApiImpl.java

```diff
@@ -0,0 +1,20 @@
+package com.icthh.xm.tmf.ms.loyalty.service;
+
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import com.icthh.xm.tmf.ms.loyalty.web.api.LoyaltyAccountApiDelegate;
+import com.icthh.xm.tmf.ms.loyalty.web.api.model.LoyaltyTransactionRef;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+@LepService(group = "tasks")
+public class LoyaltyAccountApiImpl implements LoyaltyAccountApiDelegate {
```

Comment:
Why API delegate implementation in service package?

Reply:
moved to rest